### DISCUSSION TITLE
Discord/Slack Group

### DISCUSSION THREAD

**Discussion by mayank190801** [sugarlabs/musicblocks-v4#301] [Q&A] [answered]
As a beginner who wants to join the community and start contributing, I find it a little difficult to communicate with people working on the project. 
I personally suggest we should have some Discord/Slack group, so that people can talk and share their ideas in a much more efficient manner.
It would be great if everyone can give their feedback.
Thanks!

**Comment by meganindya** [comment, marked as answer]
We have a room for Music Blocks on Matrix: [#musicblocksdev:matrix.org](https://matrix.to/#/#musicblocksdev:matrix.org)

**Reply by mayank190801** [reply]
Thanks a lot! This perfectly solves the issue.
